packages/terra-slide-panel-manager/CHANGELOG.md

```diff
@@ -3,6 +3,8 @@ Changelog
 
 Unreleased
 ----------
+### Added
+* Listens to `typeConfig` property. Currently is limited to `panelBehavior`. 
```

Comment:
From a consumer's perspective, they won't know what this means. Could we expand on this? We should also add in this will be used over the prop specified panelBahavior

Reply:
udpated here a7145805d90a12ab96589eeb0474485d9e542aa3